src/Raven.Studio/gulp/parseConfiguration.js

```diff
@@ -13,7 +13,7 @@ var HANDLER_PATTERN = "Configuration.cs";
 
 function getConfigurationConstantsFields() {
     var constantsContent =  fs.readFileSync(PATHS.configurationConstants, 'utf8');
-    var confClassStart = constantsContent.indexOf('public class Configuration');
+    var confClassStart = constantsContent.indexOf('public class Indexing');
```

Comment:
This will be handled properly by @gregolsky http://issues.hibernatingrhinos.com/issue/RavenDB-5665